<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDelayer_MainBox</class>
 <widget class="QWidget" name="PasswordDelayer_MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="password_groupBox">
     <property name="title">
      <string>Mot de passe</string>
     </property>
     <layout class="QVBoxLayout" name="password_layout">
      <item>
       <widget class="QLineEdit" name="mainLine"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hidePassword_layout">
        <item>
         <widget class="QCheckBox" name="drawPassword_checkBox">
          <property name="text">
           <string>Afficher caract√®res</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="copyToClipboard_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Copier mot de passe</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
